whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/transform/FilterAuthFunction.java

```diff
@@ -4,9 +4,12 @@
 import com.google.common.collect.Lists;
 import com.google.common.collect.Maps;
 import com.google.common.collect.Sets;
-import net.ripe.db.whois.common.oauth.OAuthSession;
+import io.netty.util.internal.StringUtil;
```

Comment:
I don't think you mean to use that class? 

You should add "internal" packages to the "Exclude from auto-import and completion" in IntelliJ.

Reply:
Talked over huddle. I am adding the internal packages into that mentioned list?